packaging/wheel/setup.py

```diff
@@ -1,11 +1,17 @@
-from setuptools import setup, Extension
+from setuptools import setup
 from setuptools.command.install import install
 from setuptools.command.sdist import sdist
 from wheel.bdist_wheel import bdist_wheel
 
+# shutil.which was added in Python 3.3
+# c.f. https://docs.python.org/3/library/shutil.html#shutil.which
+try:
+    from shutil import which
+except ImportError:
+    from distutils.spawn import find_executable as which
```

Comment:
In modernizing the Python packaging, it is preferable to use `shutil.which`, so if it is available use it. If you're on Python 2, fall back to using `distutils.spawn.find_executable`.